### PR TITLE
On macOS, add WKWebview as subview of existing NSView directly

### PR DESCRIPTION
**Pull request by wusyong**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [x] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/wry/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: src/webview/wkwebview/mod.rs [hunk 1]
```diff
@@ -663,38 +663,13 @@ r#"Object.defineProperty(window, 'ipc', {
...
-        let parent_view_cls = match ClassDecl::new("WryWebViewParent", class!(NSView)) {
-          Some(mut decl) => {
-            decl.add_method(
-              sel!(keyDown:),
-              key_down as extern "C" fn(&mut Object, Sel, id),
-            );
-
-            extern "C" fn key_down(_this: &mut Object, _sel: Sel, _event: id) {
-              // do nothing; this prevents macOS from playing an unsupported key feedback sound
-            }
```
Comment (amrbashir) [inline src/webview/wkwebview/mod.rs:675]: does this bug happen with winit and this PR?

File: src/webview/wkwebview/mod.rs [hunk 2]
```diff
@@ -663,38 +663,13 @@ r#"Object.defineProperty(window, 'ipc', {
...
-          }
-          None => class!(NSView),
-        };
-
-        // Create a view to contain the webview, without it devtools will try to
-        // inject a subview into the frame causing an obnoxious warning.
-        // See https://github.com/tauri-apps/wry/issues/273
-        let parent_view: id = msg_send![parent_view_cls, alloc];
-        let _: () = msg_send![parent_view, init];
-        parent_view.setAutoresizingMask_(NSViewHeightSizable | NSViewWidthSizable);
```
Comment (amrbashir) [inline src/webview/wkwebview/mod.rs:687]: will the removal of this disable auto resizing of the webview based on the parent? the current behavior is that the webview content will automatically fill the whole window when the window is resized.

File: src/webview/wkwebview/mod.rs (hunk 1 above)
Comment (wusyong) [reply src/webview/wkwebview/mod.rs:675]: No, TaoView/WinitView also delcared this method already.

File: src/webview/wkwebview/mod.rs (hunk 2 above)
Comment (wusyong) [reply src/webview/wkwebview/mod.rs:687]: Yes, it will be auto-resized when the window is resized.

**Review by amrbashir — APPROVED** [review]

**Pull request merged**